client/network/src/service.rs

```diff
@@ -858,6 +876,14 @@ impl Metrics {
 				"import_queue_justifications_submitted",
 				"Number of justifications submitted to the import queue.",
 			)?, registry)?,
+			incoming_connections_errors_total: register(Counter::new(
+				"sub_libp2p_incoming_connections_errors_total",
```

Comment:
```suggestion
				"sub_libp2p_incoming_connections_handshake_errors_total",
```

Incoming conncetions can as well fail after the handshake. The suggestion would align the metric name with the description. What do you think?

Reply:
The "incoming" already conveys that. In the terminology of rust-libp2p, "incoming" means "during the handshake".